### DISCUSSION TITLE
Unofficial Discord Server to Get Invited?

### DISCUSSION THREAD

**Discussion by kgzoom** [bluesky-social/atproto#1903] [How to do it?]
Hi,

I think I signed up months ago for an invite, but I don't even remember getting an email confirmation that I submitted an invite request in the first place (I don't even know what email I used). Is there an unofficial discord server that gives out invites? If not, where should I go? Reddit?

**Comment by kolgza** [comment]
There's an excess in the number of invite codes floating around. If you are already in several Discord servers, you'll probably find at least one thread for sharing Bluesky invites if you search for it.

Right now, I have five to share.
